src/steamship/utils/repl.py

```diff
@@ -1,16 +1,31 @@
 import abc
 import contextlib
 import logging
+import uuid
 from abc import ABC
-from typing import Any, Dict, List, Optional, Type, cast
+from typing import Any, Dict, List, Optional, Type, Union, cast
 
-from steamship import Block, Steamship, Task
+from steamship import Block, Steamship, SteamshipError, Task
+from steamship.agents.llms import OpenAI
 from steamship.agents.logging import AgentLogging
-from steamship.agents.schema import AgentContext, Tool
+from steamship.agents.schema import Agent, AgentContext, Metadata, Tool
 from steamship.agents.service.agent_service import AgentService
+from steamship.agents.utils import with_llm
 from steamship.data.workspace import Workspace
 from steamship.invocable.dev_logging_handler import DevelopmentLoggingHandler
 
+try:
+    from termcolor import colored  # noqa: F401
+except ImportError:
+
+    def colored(text: str, **kwargs):
+        print(text)
+
+
+# Use either "gpt-3.5-turbo-0613" or "gpt-4-0613" here.
+# Other versions of GPT tend not to work well with the ReAct prompt.
+MODEL_NAME = "gpt-4-0613"
```

Comment:
Why does this have to be defined in the REPL?  Is there a way to refactor around this? It feels really strange that the REPL would be selecting a default model.